File: MarsRoverProb/src/com/marsroverprob/parser/RoverPositionParser.java
package com.marsroverprob.parser;

import com.marsroverprob.helper.RoverHelper;
import com.marsroverprob.model.Rover;

public class RoverPositionParser implements AbstractParser {

  @Override
  public Rover parse(Rover rover, String roverInput, String maxPos) {
    return RoverHelper.constructRover(rover, roverInput, maxPos);
  }

}

File: MarsRoverProblem_C#/Source/marsroverprob/parser/RoverPositionParser.cs
using System;

using marsroverprob.parser.intf;
using marsroverprob.model;
using marsroverprob.helper;

namespace marsroverprob.parser.posImpl
{
    class RoverPositionParser : IAbstractParser
    {
        public Rover parse(Rover rover, String roverInput, String maxPos) {
            return RoverHelper.constructRover(rover, roverInput, maxPos);
        }
    }
}